test/unittests/api/v0.0.1/Projects_Test.php

```diff
@@ -1,226 +0,0 @@
-<?php
```

Comment:
I finally have an answer. It keeps failing the getprojectwithproject so I
removed it to see if it would compile. I wanted to know what does this test
do. It tests getting projects from the XML or the database?
On 24 Sep 2015 09:54, "Dave MacFarlane" <notifications@github.com> wrote:

> In test/unittests/api/v0.0.1/Projects_Test.php
> <https://github.com/aces/Loris/pull/1257#discussion_r40320552>:
>
> > @@ -1,226 +0,0 @@
> > -<?php
>
> Why is the API test deleted in this pull request?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aces/Loris/pull/1257/files#r40320552>.
>